Corona-Warn-App/src/main/res/layout/tracing_details_find_details_in_journal.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Since there is only one view here, we don't need a ViewGroup in this layout, see e.g. `trace_location_organizer_category_header`